jwt/src/dpop/htu.rs

```diff
@@ -49,9 +50,9 @@ impl From<url::Url> for Htu {
     }
 }
 
-impl ToString for Htu {
-    fn to_string(&self) -> String {
-        self.0.to_string()
+impl fmt::Display for Htu {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0)
     }
```

Comment:
Since we have `derive_more` in our dependencies anyway, we could use it here (`derive(Display)`) I guess!? It reduces boilerplate.